components/dashboard/src/projects/new-project/NewProjectRepoList.tsx

```diff
@@ -45,12 +45,9 @@ export const NewProjectRepoList: FC<Props> = ({ filteredRepos, noReposAvailable,
                                         </Button>
                                     ) : (
                                         <p className="text-gray-500 font-medium">
-                                            <a rel="noopener" className="gp-link" href={userLink(r)}>
-                                                @{r.inUse.userName}
-                                            </a>{" "}
-                                            already
+                                            Project already
                                             <br />
```

Comment:
**nitpick**: We could probably skip the break line here as the paragraph will wrap when the repository name is long. Besides, this will go away with Projects v2.